src/gltfPreviewDocumentContentProvider.ts

```diff
@@ -124,6 +125,7 @@ export class GltfPreviewDocumentContentProvider implements TextDocumentContentPr
 
         // Note that with the file: protocol, we must manually specify the UTF-8 charset.
         const content = this._mainHtml.replace('{assets}',
+            `<script type="text/x-draco-decoder" src="${this.getFilePath('engines/Draco/draco_decoder.js')}"></script>` +
```

Comment:
I'd like to replace this with a new string in the existing map.  I'll push a change here.